### ISSUE TITLE
Default value for MultiLine fields strips linebreaks

### ISSUE THREAD

**Issue by nyalloc** [deepnight/ldtk#752]
Setting a defualt value for multiline strings has a bug. Once the project is saved and opened again, the value will have all of it's line endings stripped, and will instead be loaded as a single line.

Intended data:

```
Example multiline string
This will not be loaded correctly
Line endings are gone
```

Actual data:

`
Example multiline string
This will not be loaded correctly
Line endings are gone
`

**Comment by deepnight** [comment]
I see, thanks for the precise report! This will help a LOT :)

**Comment by deepnight** [comment]
Fixed in 1.2.0
